### PR TITLE
Example apps for st.toast

### PR DESCRIPTION
**Pull request by sfc-gh-dmatthews**
Example apps to embed in st.toast documentation

## 📚 Context
Files to be committed prerelease in order for embedded apps to function at release.

## 🌐 References
[st.toast](https://www.notion.so/snowflake-corp/st-toast-1dd82ceb1fe4449087e2e892eda03de4?pvs=4)

**Contribution License Agreement**
By submitting this pull request you agree that all contributions to this project are made under the Apache 2.0 license.

### CODE REVIEW

File: python/api-examples-source/status.toast2.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import streamlit as st
+import time
+
+def cook_breakfast():
+    msg = st.toast('Gathering ingredients...')
+    time.sleep(1)
+    msg.toast('Cooking...')
+    time.sleep(1)
+    msg.toast('Ready!', icon = "🥞")
```
Comment (snehankekre) [inline python/api-examples-source/status.toast2.py:9]: This is really cool! Overwrites the message on the same toast container 🧠

File: python/api-examples-source/status.toast3.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import streamlit as st
+import time
+
+go = st.radio('Give a toast?', ['Yes','No'])
+
+if go == 'No': st.stop()
+
+with st.toast('Cheers!'):
+    time.sleep(2)
+    st.balloons()
```
Comment (snehankekre) [inline python/api-examples-source/status.toast3.py:10]: What does the `with` notation accomplish? Haven't gone through the spec in detail.

Selecting 'Yes' displays the toast, but not the balloons 🤔 This example could be revised. The other two are good to merge!

**Review by snehankekre — APPROVED** [review]
These look great, thank you! Approved pending changes to `status.toast3.py`.

File: python/api-examples-source/status.toast3.py (hunk 2 above)
Comment (sfc-gh-dmatthews) [reply python/api-examples-source/status.toast3.py:10]: Yes, this is a placeholder app for the next release, but I can remove it. (See project channel in Slack.)

**Pull request merged**